apps/src/templates/teacherDashboard/SectionRow.jsx

```diff
@@ -245,6 +245,15 @@ class SectionRow extends Component {
 
     const persistedSection = !!section.code;
 
+    let sectionCode = '';
+    if (!editing) {
+      if (section.providerManaged) {
+        sectionCode = '---';
+      } else {
+        sectionCode = section.code;
+      }
+    }
```

Comment:
Updated in https://github.com/code-dot-org/code-dot-org/pull/16557/commits/5be4ec75e916f2427002c130ac329c132e774270.